### PR TITLE
Fix typo on unique_items param of Field doc

### PR DESCRIPTION
**Pull request by vitor-amartins**
## Change Summary

Fixed param name from Field's doc.

## Related issue number

I didn't open an issue as the update was quite trivial.

## Checklist

* [x] Unit tests for the changes exist
* [x] Tests pass on CI and coverage remains at 100%
* [x] Documentation reflects the changes where applicable
* [x] `changes/<pull request or issue id>-<github username>.md` file added describing change
  (see [changes/README.md](https://github.com/samuelcolvin/pydantic/blob/master/changes/README.md) for details)
* [x] My PR is ready to review, **please add a comment including the phrase "please review" to assign reviewers**

### CODE REVIEW

**Comment by vitor-amartins** [comment]
please review

**Review by takos22 — APPROVED** [review]
LGTM, just found this typo too 😄

**Comment by RobRoseKnows** [comment]
Lol I submitted a duplicate PR #3867. Closed mine.

File: changes/3857-vitor-amartins.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Fix wrong name of parameter of Field's documentantion
```
Comment (hramezani) [inline changes/3857-vitor-amartins.md:1]: We don't need to mention this in changes. please update the PR and remove this file.

**Comment by hramezani** [comment]
please update

File: changes/3857-vitor-amartins.md (hunk 1 above)
Comment (vitor-amartins) [reply changes/3857-vitor-amartins.md]: done

**Review by hramezani — APPROVED** [review]

**Pull request merged**